streamparse/dsl/topology.py

```diff
@@ -195,14 +195,18 @@ def _spec_to_flux_dict(spec):
             shell_object = spec.component_object.shell
             flux_dict['constructorArgs'].append([shell_object.execution_command,
                                                  shell_object.script])
-            output_fields = ['NONE_BUT_FLUX_WANTS_SOMETHING_HERE']
-            if 'default' in spec.outputs:
-                output_fields = spec.outputs['default'].output_fields
-            flux_dict['constructorArgs'].append(output_fields)
-            if set(spec.outputs.keys()) - {'default'}:
-                raise TypeError('Flux does not currently support ShellBolts '
-                                'with multiple streams. Given: {!r}'
-                                .format(spec))
+            flux_dict['configMethods'] = []
+            for key in spec.outputs.keys():
+                if key == 'default':
+                    flux_dict['configMethods'].append({
+                        'name': 'setDefaultStream',
```

Comment:
Instead of using `setDefaultStream` here, I'd prefer to use the previous approach (i.e., appending the default stream's  fields to `constructorArgs`) for this case, so that this case will be compatible with more versions of Storm.